docs/style-guide-for-guide-articles.md

```diff
@@ -153,6 +153,9 @@ You can even embed relevant YouTube videos and interactive [REPL.it](https://rep
 
 Don't use emojis or emoticons in the Guide. freeCodeCamp has a global community, and the cultural meaning of an emoji or emoticon may be different around the world. Also, emojis can render differently on different systems.
 
+### Note: 
+We recommend not adding images to an article unless abosultely necessary. freeCodeCamp has no control over user-hosted images and images increase the page size. In addition, certain image hosting services might be blocked in some parts of the world
```

Comment:
That's not true. We can have images hosted on our side, if we wanted. While hotlinking is not recommended, the real reasons are listed in this comment: https://github.com/freeCodeCamp/freeCodeCamp/issues/35518#issuecomment-470838824

Reply:
Looks better now @raisedadead ?

Reply:
Hey @cmccormack, @RandellDawson can you help them with the verbiage here. I also think this need not be a note, and made an opening sentence on the section itself like so:

## Images

We do not recommend adding images to an article unless necessary. Images hurt Web Accessibility. They are difficult to maintain and get outdated. They are slow to load on poor connections and web hurt performance.

For including images under compelling reasons, if they aren't already hosted somewhere else on the web, [...]

Reply:
@raisedadead the topic of images and image hosting comes up quite often, and it seems we never really see much of a resolution.  While I agree we should avoid including images for the most part, we don't really provide details on how images should be added when they *are* appropriate.  Are there any plans to clarify that topic, like a preferred hosting site, or maybe fcc hosting?  Requesting attribution can be difficult as well, as most Contributors are not experts at determining if an image is free to use or stolen when reviewing a PR.  Perhaps only allowing from verified sources like wikimedia or creative commons would make this easier. 
I'll wait for @thecodingaviator to provide an update, if any, after your most recent comments and then provide as much feedback as possible.

Reply:
> [...] we don't really provide details on how images should be added when they are appropriate. [...]

We actually do provide some information in the style guide. That is the file being updated in this PR. https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#images

Reply:
It can be improved though.

Reply:
@cmccormack I think it's a topic worthy of discussion. @raisedadead Should we put this one aside for next month's fCC Contributor monthly chat?

Reply:
Not sure, when we can have the next chat. @thecodingaviator can you please update the guidelines as per my recommendation. I believe @cmccormack and @RandellDawson are waiting for you for a feedback post that on the verbiage.

Reply:
Alright @raisedadead making changes

Reply:
I've made changes. It is mostly what Mrugesh had suggested, I've made minor adjustments. I've also removed the line about adding REPLT.it links because they're not under fCC control and get outdated very fast